hbase-common/src/main/java/org/apache/hadoop/hbase/io/hfile/BlockType.java

```diff
@@ -31,6 +31,7 @@
  * The values in the enum appear in the order they appear in a version 2 HFile.
  */
 @InterfaceAudience.Private
+@SuppressWarnings("ImmutableEnumChecker")
```

Comment:
Does the warning specify the point of mutability? A quick scan here and I don't see any state mutations.